docs/content/en/docs/yaml-crd-ref/config.md

```diff
@@ -4,7 +4,7 @@ description: Define configuration values
 weight: 20
 ---
 
-`KeptnConfig` defines Keptn configuration values.
+`KeptnConfig` It is refers as configuartion files and the settings used to define the behaviour of the kubernetes cluster.
```

Comment:
Syntax of this sentence is way off and much more verbose than it should be.  And KeptnConfig has a couple very specific things it defines.  The behaviour of the Kubernetes cluster is mostly determined by the Helm charts.